### PR TITLE
Update cht interop README.md

### PR DESCRIPTION
**Pull request by Phil-Mwago**
# Description

[description]

medic/interoperability#[number]

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by lorerod — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,13 +16,11 @@ The components and reference information for interoperability used in this proje
 
 #### CHT
 
-The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it.
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it. Below is the standard data workflow;
```
Comment (andrablaj) [inline README.md:19]:
```suggestion
The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it. Below is the standard data workflow:
```

File: README.md [hunk 2]
```diff
@@ -22,13 +16,11 @@ The components and reference information for interoperability used in this proje
...
 
 ![](./docs/images/flow.png)
 
-This project uses OpenHIM as the middleware component with [Mediators](http://openhim.org/docs/configuration/mediators/) to do the conversion. [Outbound Push](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/outbound/) is configured to make a request to the middleware when relevant documents are created or modified in the CHT. A Mediator then creates a FHIR resource which is then routed to OpenHIM. OpenHIM routes the resource to any other configured systems.
-
-Conversely to bring data into the CHT, OpenHIM is configured to route the updated resource to the Mediator, which then calls the relevant [CHT APIs](https://docs.communityhealthtoolkit.org/apps/reference/api/) to update the document in the CHT database. This will then be replicated to users’ devices as per usual.
+See more information on the [CHT interoperability](https://docs.communityhealthtoolkit.org/apps/concepts/interoperability/) page on the doc site.
```
Comment (andrablaj) [inline README.md:23]:
```suggestion
See more information on the [CHT interoperability](https://docs.communityhealthtoolkit.org/apps/concepts/interoperability/) page on the CHT documentation site.
```

**Review by andrablaj — CHANGES_REQUESTED** [review]
@Phil-Mwago There is one technical paragraph I suggest we keep in the `README` to explain the architecture flow. The other suggestions are minor.

File: README.md (hunk 2 above)
Comment (andrablaj) [reply README.md:23]: I suggest being extra clear about the documentation site, as some readers might not be aware of what `doc` site might mean.

File: README.md [hunk 3]
```diff
@@ -22,13 +16,11 @@ The components and reference information for interoperability used in this proje
 
 #### CHT
 
-The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it.
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it. Below is the standard data workflow;
 
 ![](./docs/images/flow.png)
 
-This project uses OpenHIM as the middleware component with [Mediators](http://openhim.org/docs/configuration/mediators/) to do the conversion. [Outbound Push](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/outbound/) is configured to make a request to the middleware when relevant documents are created or modified in the CHT. A Mediator then creates a FHIR resource which is then routed to OpenHIM. OpenHIM routes the resource to any other configured systems.
```
Comment (andrablaj) [inline README.md:29]: Super weird, my comments from yesterday disappeared; I'm not sure why. I saw some changes in the GitHub interface this morning that might be connected. 🤔 

I think you should keep this paragraph as it explains nicely (and in a technical way) the architecture image above. I don't find it redundant nor overly complicated.

Comment (Phil-Mwago) [reply README.md:29]: Done

File: README.md [hunk 4]
```diff
@@ -22,13 +16,13 @@ The components and reference information for interoperability used in this proje
...
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it. Below is the standard data workflow:
 
 ![](./docs/images/flow.png)
 
 This project uses OpenHIM as the middleware component with [Mediators](http://openhim.org/docs/configuration/mediators/) to do the conversion. [Outbound Push](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/outbound/) is configured to make a request to the middleware when relevant documents are created or modified in the CHT. A Mediator then creates a FHIR resource which is then routed to OpenHIM. OpenHIM routes the resource to any other configured systems.
 
-Conversely to bring data into the CHT, OpenHIM is configured to route the updated resource to the Mediator, which then calls the relevant [CHT APIs](https://docs.communityhealthtoolkit.org/apps/reference/api/) to update the document in the CHT database. This will then be replicated to users’ devices as per usual.
```
Comment (andrablaj) [inline README.md:31]: As discussed in this comment, I believe this paragraph can stay, too, as it continues the paragraph above.

**Review by andrablaj — CHANGES_REQUESTED** [review]
Looks good, one more tiny suggestion!

**Review by andrablaj — APPROVED** [review]
👌 Looks good, thank you @Phil-Mwago!

**Comment by Phil-Mwago** [comment]
> Looks good, one more tiny suggestion!



> 👌 Looks good, thank you @Phil-Mwago!

Thanks @andrablaj

**Pull request merged**